.gitignore

```diff
@@ -1,3 +1,4 @@
 bin/
 vendor/
 composer.lock
+.phpunit.result.cache
```

Comment:
I'd prefer this to be added in developer's global `.gitignore`.

Reply:
WFM! :+1: